packages/web-app/src/views/questionnaire-check-and-confirm.njk

```diff
@@ -0,0 +1,33 @@
+{% extends "layouts/main.njk" %}
```

Comment:
This template should extent the form layout

Reply:
It still needs to do a POST though because AC2 has it updating the status and updating the record in the database.

Reply:
Fair enough